### PR TITLE
[Fibonacci] added works

### PR DESCRIPTION
**Pull request by Crazyayane07**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @Crazyayane07

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Crazyayane07 for on-time delivery

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,31 @@
 #pragma once
 
-int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+int fibonacci_iterative(int sequence)
+ {
+	if (sequence < 2)
+	{
+		return sequence;
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:7]: Wrong indentation

File: homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
+	}
+
+	int result = 1;
+	int prevResult = 0;
+
+	for (int i = 1; i < sequence; i++) 
+	{
+		int tempResult = result;
+		result += prevResult;
+		prevResult = tempResult;
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:17]: Wrong indentation (should be 4 spaces)

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**